### DISCUSSION TITLE
Add Line Ruler for line of sight and combat

### DISCUSSION THREAD

**Discussion by Daza99** [luiscastilho/dungeoneering#13] [Ideas]
Like we press and hold spacebar and then click where we want the line to originate from then move the mouse to the destination. This also could be handy for combat shooting a gun, bow, casting a spell and seeing if in a straight line it would hit the target.  This feature also could be used to see if an enemy, like a sentry or a turret would see the party or character approach.  

I see there is a Sight types in the menu for the token, but it looks like this is just tracking the peripheral vision of the character and has no representation on the map itself.

**Comment by luiscastilho** [comment]
Good idea! I will add this to the TODO list. And considering a cell is a 5x5 ft. space, this ruler could show the distance from one point to another. Related to that but not exactly, I would also like to show how a far a token is being moved as you drag it from one cell to another.

The sight types are used with dynamic lighting, to reveal parts of the map. If a map is dark and a token has Darkvision 30', it will reveal 30' around it even if it doesn't have a light source assigned to it. But yeah, it's not related to line of sight.
